Source/Scene/parseKTX2.js

```diff
@@ -0,0 +1,225 @@
+import arraySlice from "../Core/arraySlice.js";
```

Comment:
`parseKTX2` and `KTX2Transcoder` belong in `Core` since they are called from `loadKTX2` which is in `Core`